packages/broker/src/ConfigWizard.ts

```diff
@@ -171,7 +165,17 @@ Object.keys(PLUGIN_DEFAULT_PORTS).map((pluginName) => {
     })
 })
 
-prompts = prompts.concat(pluginSelectorPrompt).concat(pluginPrompts)
+const revealGeneratedPrivateKeyPrompt = {
+    type: 'confirm',
+    name: 'revealGeneratedPrivateKey',
+    message: 'We strongly recommend backing up your private key. It will be written into the config file, but would you also like to see it on screen now?',
```

Comment:
```suggestion
    message: 'We strongly recommend backing up your private key. It will be written into the config file, but would you also like to see this sensitive information on screen now?',
```

Reply:
Small text edit to emphasize the sensitivity of the information